pkg/test/step_test.go

```diff
@@ -287,8 +288,11 @@ func TestRun(t *testing.T) {
 					}),
 				},
 			}, func(t *testing.T, client client.Client) {
+				pod := testutils.NewPod("hello", testNamespace)
+				assert.Nil(t, client.Get(context.TODO(), types.NamespacedName{Namespace: testNamespace, Name: "hello"}, pod))
```

Comment:
How is this related to the bump?

Reply:
I'm not fully sure, but if it's not in there the test fails on Line 295 with "object was modified". So fetching it once before fixes that.